src/components/TeamPage/teamPhotos.js

```diff
@@ -60,14 +60,17 @@ const ProfileRole = styled.span`
   max-width: 150px;
 `
 
-const ProfileImg = styled(Img)`
+const ProfileImg = styled(GatsbyImage)`
   picture img {
     transition: filter ease-in 200ms !important;
   }
 `
 
 const ProfileImgWrapper = styled.div`
   position: relative;
+  display: flex;
+  justify-content: center;
+  margin-bottom: 4.2px;
```

Comment:
I think this is where the random 4.2 margin is coming from 🔍

Reply:
oh yeah haha i added this in because it was not centered and was missing 4.2px before